src/governance/object.h

```diff
@@ -30,10 +30,10 @@ extern RecursiveMutex cs_main;
 static constexpr double GOVERNANCE_FILTER_FP_RATE = 0.001;
 
 
-static constexpr CAmount GOVERNANCE_PROPOSAL_FEE_TX = (1 * COIN);
+static constexpr CAmount GOVERNANCE_COMMITMENT_AMOUNT = (1 * COIN);
+static constexpr int64_t GOVERNANCE_COMMITMENT_CONFIRMATIONS = 6;
```

Comment:
@UdjinM6, can you explain the rationale/purpose for the 1 and 6 block confirmations here? Does it make more sense now to require InstantSend and/or ChainLocks for proposal validity instead of block confirmations (in a separate issue/PR)?

Reply:
We had no DML (InstantX wasn't as reliable as deterministic InstantSend) and no ChainLocks (reorgs up to 6 blocks could happen easily) back then. We could indeed consider proposals that have their commitments IS-/ChainLocked safe to accept without waiting for 6 confs now. I'd probably still require at least 1 conf to keep the logic simple because handling "orphan" proposals can be complicated (error/bug-prone) imo.